src/js/leaflet.storage.xhr.js

```diff
@@ -66,9 +68,10 @@ L.Storage.Xhr = {
                     L.Storage.fire('ui:alert', {content: msg, level: 'error', duration: 100000, actions: actions});
                 }
                 else {
-                    if (xhr.status !== 0) {  // 0 === request cut by user
-                        L.Storage.fire('ui:alert', {'content': L._('Problem in the response'), 'level': 'error'});
-                    }
+                    var errorMessage = L._('HTTP error {errorcode}', {errorcode: xhr.status});
```

Comment:
Adding the HTTP error code sounds a good idea, but it would put it as a secondary information, keeping the previous message, something like: 'Problem in the response (error code: xxx)'. We don't need to add it's an *HTTP* error code, because normal people doesn't know what HTTP is. Though, having the code itself can help while trying to debug and communicate with more technical users.